controls/pdfviewer/overview.md

```diff
@@ -14,9 +14,15 @@ previous_url: pdfviewer-overview, pdfviewer-manipulating-programatically
 
```

Comment:
This article can be more informative. We could add the following, for example:
* A paragraph with a link to the Getting Started for the specific control.
* Next Steps section is also an option.
* The best features of the specific control
* How it works (high-level)
* The common scenarios to use this control